src/locales/en.json

```diff
@@ -4,67 +4,67 @@
     "about": "About",
     "spreadsheet": "Spreadsheet",
     "contactus": "Contact Us",
-    "whitepaper": "Whitepaper"
+    "whitepaper": "White Paper"
   },
   "current_lang_two_letter_code": "en",
   "calculator": {
     "precautions": {
       "label": "Precautions",
       "environment_header": "Environment",
       "environment_question": "Is it indoor or outdoor?",
-      "no_intimate_bonus_outdoors": "Due to very close distances, we are not confident that being outdoors reduces the risk in a substantial way. Thus, we are not providing any bonus for being outdoors when intimate.",
+      "no_intimate_bonus_outdoors": "Due to very close distances, being outdoors probably doesn't reduce the risk substantially. There is no bonus for being outdoors when intimate.",
       "your_mask_header": "Your mask",
       "your_mask_question": "What mask are YOU wearing?",
       "your_mask_note": "If you’re eating or drinking, select “no mask”",
       "their_mask_header": "Their mask",
       "their_mask_question": "What mask are THEY wearing?",
       "their_mask_note": "If they’re eating or drinking, select “no mask”",
-      "filtered_ensure_airflow": "Ensure sufficient flow through the filter for your room. For help on how to do this, see <1>here</1>.",
+      "filtered_ensure_airflow": "Ensure sufficient airflow through the filter for your room, <1>like so</1>.",
       "volume_header": "Volume",
       "volume_question": "How much is everyone talking?"
     },
-    "risk_note_about_household_members": "When estimating your risk of infection from a household member or partner/spouse we assume these interactions are indoors, unmasked, and undistanced.",
+    "risk_note_about_household_members": "Your estimated risk of infection from a household member or partner/spouse assumes these interactions are indoors, unmasked, and undistanced.",
     "explanationcard": {
       "person": "person",
       "person_plural": "people",
       "multiple_suffix": "x your weekly risk budget",
       "percentage_suffix": "% of your weekly risk budget",
-      "details_header_closed": "Show calculation details",
-      "details_header_open": "Hide calculation details",
+      "details_header_closed": "Show details",
```

Comment:
Help me understand what is better about this? (... I am going to leave this comment often, I expect 🙂)

Reply:
Because it is already clear what it is. Referencing _calculation_ details in what I think is a calculator setting doesn't add anything.